simtools/io_handler.py

```diff
@@ -85,16 +87,14 @@ def get_output_directory(self, label=None, dir_type=None, test=False):
         if self.use_plain_output_path:
             path = Path(self.output_path)
         else:
-            if test:
-                output_directory_prefix = Path(self.output_path).joinpath("test-output")
-            else:
-                output_directory_prefix = Path(self.output_path).joinpath("simtools-output")
-
-            today = datetime.date.today()
-            label_dir = label if label is not None else "d-" + str(today)
+            output_directory_prefix = Path(self.output_path).joinpath(dir_type[:8] + "-output")
```

Comment:
Could you please indicate what the actual issue is? Not liking it is not enough for me to work with.

Reply:
I thought that was clear, but I can rephrase it. Since I can pass any name in `dir_type` I like, I could choose e.g. 'applications-prod-5-southern'-array. With that my subdirectory would be `applicat-output`, i.e., this is not the name I passed and the string was truncated without any apparent reason. Due to this issue, my main suggestion is to constrain the choices of `dir_type` to a selection of allowed names, as I mentioned before.

Reply:
And I would add that we don't have to constrain, but just remove `simtools-` using regex if this is specifically your original goal here @GernotMaier (I must admit I couldn't figure out the reasoning behind it yesterday).

Reply:
You have a suggestion on the regex expression?

Reply:
Given below (with another optional solution). 
And BTW, reading my comment again made me realise that even here I still did not understand what you wanted to do until I had to do it myself.

Reply:
OK - now I get what the issue is. Clearly missed it, as I was testing with existing use cases only. You need to make comments in review clear enough, I obviously would have done it differently if I would have been aware of it.

@orelgueta - thanks, I've implemented the solution.

I think we shouldn't fix the options of dir_type, unless there is a logical reason for it.

Reply:
OK, so if you implemented the regex solution, please add also a mention in the docstring.